### PR TITLE
Fixing some language issues on logging.md (FR)

### PR DESCRIPTION
**Pull request by didier-durand**
Hello,
as said in title: fixing some language issues on logging.md (FR)
Didier

### CODE REVIEW

File: content/fr/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -12,18 +12,18 @@ weight: 60
...
 comprendre ce qui se passe dans un cluster. Les journaux sont particulièrement
 utiles pour débogguer les problèmes et surveiller l'activité du cluster. La
-plupart des application modernes ont un mécanisme de journalisation
+plupart des applications modernes ont un mécanisme de journalisation
 d'évènements, et la plupart des environnements d'exécution de conteneurs ont été
 conçus pour supporter la journalisation des évènements. La méthode de
 journalisation la plus facile et la plus répandue pour des applications
 conteneurisées est d'écrire dans les flux de sortie standard et d'erreur
-standard (`stdout` et `stderr`).
+standards (`stdout` et `stderr`).
```
Comment (perriea) [inline content/fr/docs/concepts/cluster-administration/logging.md:20]:
```suggestion
(`stdout` et `stderr`).
```
Je pense pas qu'il utile de faire une repetition

**Review by perriea — CHANGES_REQUESTED** [review]

**Comment by didier-durand** [comment]
@perriea : committed your suggested changes

**Pull request merged**